index.js

```diff
@@ -959,14 +979,9 @@ module.exports = {
         return response
     },
 
-    deleteObject: async function (allocation_id, path, client_json) {
-        const url = proxyServerUrl + Endpoints.PROXY_SERVER_DELETE_ENDPOINT
-        const formData = new FormData();
-        formData.append('allocation', allocation_id);
-        formData.append('remote_path', path);
-        formData.append('client_json', JSON.stringify(client_json));
-        const response = await utils.delReq(url, formData);
-        return response
+    deleteObject: async function (allocation_id, path, commit) {
+        const resp = await goWasm.sdk.delete(allocation_id, path, commit);
```

Comment:
When reading 'commit' parameter, I soon thought of a hash.
Took me several seconds to remember that this is a flag.
The variable name should reflect that, much easier to understand the method.

```suggestion
    deleteObject: async function (allocation_id, path, shouldAutoCommit) {
        const resp = await goWasm.sdk.delete(allocation_id, path, shouldAutoCommit);
```